common/src/types/v0/transport/nvme_nqn.rs

```diff
@@ -74,6 +79,14 @@ impl NvmeNqn {
         let uuid = uuid::Uuid::new_v4();
         Self::Unique { uuid }
     }
+    /// Generate a Mayastor OpenEbs type name.
+    pub fn new_ebs_name(name: &str) -> Self {
```

Comment:
nit: `new_org_name()` as this would be org type I believe?

Reply:
yes that's better :+1: